app/models/model.py

```diff
@@ -204,15 +55,83 @@ def check_num_items_holes(cls, v):
         for pt_list in v:
             for pt in pt_list:
                 if len(pt) != 3:
-                    raise ValidationError(
+                    raise ValueError(
                         'Number of floats must be 3.'
                     )
         return v
 
 
-class Ground(BaseModel):
+class ShadeEnergyPropertiesAbridged(BaseModel):
 
-    type: Enum('Ground', {'type': 'Ground'})
+    type: Enum('ShadeEnergyPropertiesAbridged', {
+               'type': 'ShadeEnergyPropertiesAbridged'})
+
+    diffuse_reflectance: float = Schema(
+        0.2,
+        ge=0,
+        le=1
+    )
+
+    specular_reflectance: float = Schema(
+        0,
+        ge=0,
+        le=1
+    )
+
+    transmittance: float = Schema(
+        0,
+        ge=0,
+        le=1
+    )
+
+    transmittance_schedule: str = Schema(
+        default=None,
+        regex=r'[A-Za-z0-9_-]',
+        min_length=1,
+        max_length=100
+    )
+
+    @validator('transmittance_schedule')
+    def check_field(cls, v, values):
+        if values['transmittance'] != "" and v != "":
+            raise ValueError(
+                'Can not specify both transmittance and transmittance_schedule.'
+            )
+
+
+class ShadePropertiesAbridged(BaseModel):
```

Comment:
Is there a `ShadeProperties` that is not abridged?

Reply:
Yes, and see above.  When you find the properties here as an `Abridged` version, it means that both energy and radiance properties under this tag are abridged.  So it also means things like the radiance material under `ShadeRadianceProperties` is just the name of the material and not a full Radiance material dictionary.